src/main/java/io/weaviate/client/v1/async/users/api/RoleRevoker.java

```diff
@@ -20,8 +20,16 @@ public class RoleRevoker extends AsyncBaseClient<Boolean> implements AsyncClient
   private String userId;
   private List<String> roles = new ArrayList<>();
 
+  private final String _userType;
```

Comment:
question: why do we need to use underscores in class field names?

Reply:
is it bc of line 48?

Reply:
I wanted the `class Body` to re-use that field from the outer `RoleRevoker` and without the underscore the names were clashing.

Plus I thought that the underscore would kind of underline the very "internal" nature of this variable -- not something to be interacted with at all.

Reply:
It doesn't have to be that, we could also pass that to `new Body()` explicitly when building a request. Just seemed like a good thing to set once on instantiation.

Reply:
ok, thanks for clarification